server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader9to10.java

```diff
@@ -133,6 +141,40 @@ public void upgradeZookeeper(ServerContext context) {
     createScanServerNodes(context);
   }
 
+  private static final AtomicBoolean asyncErrorOccurred = new AtomicBoolean(false);
+
+  private void validateACLs(ServerContext context) {
+
+    final ZooReaderWriter zrw = context.getZooReaderWriter();
+    final ZooKeeper zk = zrw.getZooKeeper();
+    final String rootPath = context.getZooKeeperRoot();
+    try {
+      ZKUtil.visitSubTreeDFS(zk, rootPath, false, (rc, path, ctx, name) -> {
+        try {
+          final Stat stat = new Stat();
+          final List<ACL> acls = zk.getACL(path, stat);
+
+          if (((path.equals(Constants.ZROOT) || path.equals(Constants.ZROOT + Constants.ZINSTANCES))
+              && !acls.equals(ZooDefs.Ids.OPEN_ACL_UNSAFE))
+              || (!ZooUtil.PRIVATE.equals(acls) && !ZooUtil.PUBLIC.equals(acls))) {
+            log.error("ZNode at {} has unexpected ACL: {}", path, acls);
+            asyncErrorOccurred.set(true);
+          } else {
+            log.info("ZNode at {} has expected ACL.", path);
```

Comment:
Should this be at trace level - other than development it does not seem that I would care the path had the correct ACls